src/utils/dateRange.ts

```diff
@@ -51,3 +51,21 @@ export function getSinceDateRangeString(key: string = 'since_date') {
     startDate,
   });
 }
+
+export function getMonthDate(index: number) {
+  const today = new Date();
+  const year = today.getFullYear();
+  const month = today.getMonth() - index;
+  const startDate = new Date(year, month, 1);
+  const endDate = index === 0 ? today : new Date(year, month + 1, 0);
```

Comment:
I don't believe we want to use today's `year` here. For example, if the current month is January 2021, the previous month would be December 2020.

Reply:
change index to offset